C/c4.def

```diff
@@ -110,6 +110,7 @@ c4blob_keyFromString
 c4blob_keyToString
 c4blob_openStore
 c4blob_deleteStore
+c4blob_freeStore
```

Comment:
Please only add these below the `; C4Tests` line.  You can rename this comment if you like but I use all lines above this for the C# bindings script to generate bindings only for what is needed.